lib/pos_consensus_controller.go

```diff
@@ -211,3 +211,109 @@ func (cc *ConsensusController) HandleHeader(pp *Peer, msg *MsgDeSoHeader) {
 func (cc *ConsensusController) HandleBlock(pp *Peer, msg *MsgDeSoBlock) {
 	// TODO
 }
+
+// processNewTipBlock tries to apply a new tip block to both the Blockchain and FastHotStuffEventLoop data
+// structures. It guarantees
```

Comment:
Finish comment? also try to start the comment w/ the function name like `tryProcessBlockAsNewTip tries to apply...`